setup.py

```diff
@@ -117,7 +117,7 @@ def run(self):
                         'indy-crypto==0.4.5',
                         'python-rocksdb==0.6.9',
                         'python-dateutil==2.6.1',
-                        'pympler==0.5',
+                        'pympler==0.8',
```

Comment:
I believe it needs to be specified in build-scripts/ubuntu-1604/build-3rd-parties.sh as well (and in indy-node's repo too).

Reply:
Fixed